commands/actions/publish.go

```diff
@@ -91,11 +91,11 @@ func buildFunc(cmd *cobra.Command, args []string) {
 	}
 
 	util.MustExistDir(actions.Sources)
-	if actions.Runtime != actionsModel.RuntimeV1 {
+	if !actionsModel.IsRuntimeSupported(actions.Runtime) {
 		logrus.Error(commands.Colorizer.Sprintf(
 			"Configured runtime '%s' is not supported. Supported values: {%s}",
 			commands.Colorizer.Bold(commands.Colorizer.Red(actions.Runtime)),
-			commands.Colorizer.Bold(commands.Colorizer.Green(actionsModel.RuntimeV1)),
+			commands.Colorizer.Bold(commands.Colorizer.Green("'"+strings.Join(actionsModel.SupportedRuntimes, "','")+"'")),
```

Comment:
Do we need the `'` here? It might cause someone to try to input the runtime as `'V1'` instead of `V1`.